core/jazz_create-serverless-service/index.js

```diff
@@ -232,7 +232,8 @@ var getServiceData = (service_creation_data, authToken, configData, deploymentTa
             "REGION": service_creation_data.region,
             "USERNAME": user_id,
             "IS_PUBLIC_ENDPOINT": service_creation_data.is_public_endpoint || false,
-            "STATUS": "creation_started"
+            "STATUS": "creation_started",
+            "PLATFORM": service_creation_data.platform
```

Comment:
Since `platform` is required, add validation for the field similar to other required fields in service metadata.

Reply:
I will take this